docs/app/references/migration-guide.mdx

```diff
@@ -68,6 +68,44 @@ For users with the existing `experimentalJustInTimeCompile` flag set, you can re
 
 :::
 
+### Changes to `cy.origin()`
+
+To support sites that use [origin-keyed agent clusters](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin-Agent-Cluster), Cypress no longer injects `document.domain` into `text/html` content by default.
```

Comment:
I'd like to mention the document.domain deprecation somewhere here: https://developer.chrome.com/blog/document-domain-setter-deprecation - just want to give context that these are outward changes enforcing this direction.

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/5974/commits/644aa612b9a136472c6f1be955f49458f3770b70